# received command, but robot is not listening and ignoring commands
string command_received
